docs/api/paths/admin/_all_dbs.yaml

```diff
@@ -8,32 +8,25 @@
 get:
   summary: Get a list of all the databases
   description: |-
-    This retrieves all the database's names that are in the current Sync Gateway node.
+    This retrieves all the databases that are in the current Sync Gateway node. If verbose, return bucket and status information about the databases, otherwise return the names.
```

Comment:
```suggestion
    This retrieves all the databases that are in the current Sync Gateway node. If verbose, returns bucket and state information for each database, otherwise returns names only.
```